content/academy/puppeteer_playwright/collecting_data.md

```diff
@@ -58,23 +58,27 @@ await browser.close();
 </marked-tab>
 ```
 
-> Notice the slightly different syntax between Playwright and Puppeteer with `waitForLoadState('networkidle')` and `waitForNetworkIdle()`. Sometimes, the differences are fairly subtle like this, but later on we'll run into some more significant differences.
+> Notice the slightly different syntax between Playwright and Puppeteer with `waitForLoadState('DOMContentLoaded')` and `waitForNavigation({ waitUntil: 'DOMContentLoaded' })`. Sometimes, the differences are fairly subtle like this, but later on we'll run into some more significant differences.
```

Comment:
This is probably redundant because `page.goto({ waitUntil: ...})` is the same in both.